src/pages/Stats.jsx

```diff
@@ -0,0 +1,260 @@
+      }
+    }
+    //console.log('final:', final)
+    //need to add code to sort volunteer names alphabetically
+
+    this.setState({
+      ...this.state,
+      schedules: final
+    });
+  }
+
+  //searching for volunteer names in table
+  searchVolunteers() {
+
+    var input, filter, table, tr, td, i, txtValue;
+    table = document.getElementById("myTable");
+    tr = document.getElementsByTagName("tr");
+    input = document.getElementById("myInput")
+    filter = this.state.search.toUpperCase();
+    
+    //filtering table based on characters inputted in search in real-time
+    if(this.state.search != null) {
+      for (var char = 0; char < tr.length; char++) {
+        td = tr[char].getElementsByTagName("td")[0];
+        if (td) {
+          txtValue = td.textContent || td.innerText;
+          if (txtValue.toUpperCase().indexOf(filter) > -1) {
+            tr[char].style.display = "";
+          } else {
+            tr[char].style.display = "none";
+          }
+        }
+      }
+    }
+  }
+
+  render() {
+
+    let content;
+    const {schedules} = this.state
+
+    if (schedules === null) {
+      content = (<Loading/>);
+    } else {
+
+      content = (
+        <div className='stats'>
+          <div className='page-wrapper'>
+            <TitleSection
+              title={strings.statsTitle}
+              description={strings.statsDescription}
+            />
+            <div>
+                <input type="text" 
+                id="myInput"
+                placeholder="Search a name..."
+                name="search"
+                value={this.state.search}
+                onInput={(e) => {
+                  this.setState({search: e.currentTarget.value})
+                }}
+              />
+              {/* calling search function */}
+              {this.searchVolunteers()}
+              <table className='Volunteers' id="myTable">
+                <thead>
+                  <tr>
+                    <th>Volunteer Name</th>
+                    <th>Declined 3</th>
+                    <th>Declined 2</th>
+                    <th>Declined 1</th>
+                  </tr>
+                </thead>
+                {this.state.schedules.map((val, i) => {
+                  return (
+                    <tbody key={i}>
+                      <tr>
+                        <td>{val.name}</td>
+                        {val.dates.map((date, dateindex) => {
+                          return (
+                            <td key={dateindex}>
+                              {moment(date['updatedAt']).format("MMM D, YYYY")}
+                            </td>
+                          )
+                        })}
+                      </tr>
+                    </tbody>
+                  );
+                })}
+              </table>
+            </div>
+          </div>
+        </div>
+      )
+    }
+
+    return (
+      <div>
+        <HeaderBar
+          // goBack={this.props.history.goBack}
```

Comment:
you can just remove this code. Since this page isnt coming from a link, users will only come from the url

Reply:
The back arrowLeftWhite button is apart of the HeaderBar though. Removing this will make the button useless. Is there a way to hide the button only for this page?

Reply:
can make an update to nice the back arrow if the prop is not passed it